### PR TITLE
Enable gifting on christmas

### PR DESCRIPTION
**Pull request by benthecarman**
thought this would be fun

### CODE REVIEW

**Review by futurepaul — COMMENTED** [review]
lol nice

File: src/i18n/en/translations.ts [hunk 1]
```diff
@@ -459,6 +459,7 @@ export default {
         gift: {
             give_sats_link: "Give sats as a gift",
             title: "Gifting",
+            no_plus_caption: "Upgrade to Mutiny+ to enabled gifting",
```
Comment (futurepaul) [inline src/i18n/en/translations.ts:462]:
```suggestion
            no_plus_caption: "Upgrade to Mutiny+ to enable gifting",
```

**Review by futurepaul — APPROVED** [review]
just noticed this typo and it's on the existing version so might be nice to get this in

**Comment by benthecarman** [comment]
Fixed 2 more typos my IDE found

**Pull request merged**